src/components/tech/JoinTeam.tsx

```diff
@@ -0,0 +1,57 @@
+import { Button, Flex, Heading, Section, Text, Link } from "@radix-ui/themes";
+import Image from "next/image";
+import React from "react";
+
+export const JoinTeam = () => {
+  return (
+    <Section
+      mx={{ md: "94px", initial: "2" }}
+      className="bg-[#98BEC6] rounded-3xl"
+      px={{ md: "9", initial: "6" }}
+      py="6"
+    >
+      <Flex
+        gap="6"
+        direction={{ lg: "row", initial: "column" }}
+        justify="center"
+        align="center"
+      >
+        <Flex gap="2" direction="column" justify="center">
+          <Heading size="8" className="text-navy-800">
+            How To Join Our Team!
+          </Heading>
+          <Heading size="5">
```

Comment:
**suggestion (blocking):** This isn't really a heading, just emphasized text, so use the [Strong](https://www.radix-ui.com/themes/docs/components/strong) component here or the Text component with appropriate font-weight.